asp_interactive_configuration/ooasp/interactive.py

```diff
@@ -674,7 +769,7 @@ def select_object_class(self,object_id:int,object_class:str)->None:
             raise e
 
 
-    def extend_domain(self,num:int=1,cls='object',propagate=False)->None:
```

Comment:
At least one call to this method (in Interactive.ipynb) has not been updated, leading to a `TypeError: extend_domain() got an unexpected keyword argument 'propagate'`. Please make sure to update calls when changing method signatures.

Reply:
This still does not work in Interactive.ipynb.